docs/se/openapi-geneator.adoc

```diff
@@ -0,0 +1,218 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
Many if not all of the comments I had for the MP page will apply to the SE page as well. I'm not going to repeat all of them but the corresponding changes should be made to this page also.

In fact, we should look at putting the common content into a separate file and including it into the separate MP and SE pages. I can look at doing that myself a little later.